.github/workflows/reusable-upgrade-latest-test.yaml

```diff
@@ -61,10 +46,13 @@ jobs:
       - uses: actions/checkout@v3
       - run: |
           git fetch --prune --unshallow
-      - name: Download latest release
-        uses: actions/download-artifact@v3
+      - uses: robinraju/release-downloader@v1.8
```

Comment:
YES. This may also fix https://github.com/kairos-io/kairos/issues/1601 as I found out that seems faster to download the release directly when needed!